layouts/user/content.html

```diff
@@ -0,0 +1,21 @@
+<div class="td-content">
+	<h1>{{ .Title }}</h1>
+	{{ with .Params.description }}<div class="lead">{{ . | markdownify }}</div>{{ end }}
+	{{ .Content }}
+  <hr />
+  Still have questions?  No problem!  Just head over to the
+<a href="https://www.zen-cart.com/forum.php">
+  main Zen Cart support forum</a> and ask there.
```

Comment:
Did you mean to code this text directly into the template? Or just call a shortcode from it?

Reply:
You can't call a shortcode in a template.  That's why I had to duplicate the content of faq_questions.html.